Bonsai.System/IO/GetEnvironmentVariable.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Bonsai;
+using System;
+using System.ComponentModel;
+using System.Linq;
+using System.Reactive.Linq;
+
+namespace Bonsai.IO
+{ 
+    /// <summary>
+    /// Represents an operator that returns the value of an OS environment variable.
```

Comment:
More generally, environment variables are specified per-process, so they might be different than OS variables. I would probably try to keep the wording of the original docs and say something like:

```c#
/// Represents an operator that gets the value of an environment variable for the current process.
```